kobo/apps/openrosa/apps/logger/migrations/0044_add_uid_to_attachment.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations, models
+
+from kpi.fields.kpi_uid import KpiUidField, UUID_LENGTH
+
+
+def add_uid_to_attachment(apps, schema_editor):
+    Attachment = apps.get_model('logger', 'Attachment')
+    batch_size = 2000
+    qs = Attachment.objects.only('uid').filter(uid='')
+    batch = []
+    for attachment in qs.iterator(chunk_size=batch_size):
+        attachment.uid = KpiUidField.generate_unique_id('at')
+        batch.append(attachment)
+        if len(batch) >= batch_size:
+            Attachment.objects.bulk_update(batch, ['uid'])
+            batch = []
+    if batch:
+        Attachment.objects.bulk_update(batch, ['uid'])
+
+
+def noop(*args, **kwargs):
+    pass
+
```

Comment:
No, let's not used this concept anymore. On production, it may take a while to backfill. 
See TASK-1534 for more details.